com.github.coslyk.MoonPlayer.json

```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "com.github.coslyk.MoonPlayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "moonplayer",
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--device=all",
+      "--share=network",
+      "--socket=pulseaudio",
+      "--filesystem=xdg-videos",
+      "--filesystem=xdg-download",
+      "--env=LC_NUMERIC=C"
+  ],
+  "modules": [
```

Comment:
I'm not convinced that nesting modules so many times increases readability.

Reply:
I just copied the code for building libmpv from [here](https://github.com/flathub/io.github.GnomeMpv/blob/master/io.github.GnomeMpv.json)...